src/wokwi_client/client.py

```diff
@@ -233,3 +240,23 @@ async def set_control(
             value: Control value to set (float).
         """
         return await set_control(self._transport, part=part, control=control, value=value)
+
+    async def framebuffer_read(self, id: str) -> ResponseMessage:
+        """Read the current framebuffer for the given device id."""
+        return await framebuffer_read(self._transport, id=id)
+
+    async def framebuffer_png_bytes(self, id: str) -> bytes:
```

Comment:
Let's keep the naming convention consistent. If other methods are called "save_framebuffer_...", then this one should be "get_framebuffer_png_bytes" (or read_...)